src/test/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeTest.java

```diff
@@ -410,6 +410,48 @@ public void trackPurchaseWithoutProducts() {
     Analytics.trackAction("purchase", contextData);
   }
 
+  @Test
+  public void videoPlaybackDelegatePlay() {
+    newVideoSession();
```

Comment:
Do we need to do this whole setup here? Seems like we could just call new PlaybackDelegate and only test that bit.